templates/patient/partials/portal.html.twig

```diff
@@ -33,7 +33,7 @@ The Patient Portal card for the Medical Record Dashboard
     {% if isPortalAllowed == false %}
         <div class="alert alert-warning" role="alert">
             <p class="font-weight-bold">{{  "Portal Access"|xlt }}</p>
-            <p> {{ "Allow Patient Portal in Demographics Choices."|xlt }} </p>
+            <p> {{ allowpp }} </p>
```

Comment:
```
<p> {{ "Allow Patient Portal in Demographic"|xlt }} {{ allowpp|xlLayoutLabel|text }} </p>
```

Note need to add a new xlLayoutLabel function in src/Common/Twig/TwigExtension.php (would put it above the xlListLabel function):
```
            new TwigFilter(
                'xlLayoutLabel',
                function ($string) {
                    return xl_layout_label($string);
                }
            ),
```

Reply:
many thanks @bradymiller - have implemented it the way you suggest in the twig template.